### DISCUSSION TITLE
speckled colors

### DISCUSSION THREAD

**Discussion by lyricsdude** [gopro/labs#1113] [General]
I switched from shooting WIDE=1 to normal flat colors. But on my Gopro 11 saturated objects that i film are often speckled. How to deal with this? Is this fixed on the Gopro 13? Or is there a way to shoot in wider flat colors?

**Comment by dnewman-gpsw** [comment]
Switched WIDE to normal.  So you are not wanting a wild gamut?

**Reply by lyricsdude** [reply]
yes but there's no accurate transform for WIDE=1. The technical LUTs and davinci plugins the colors are too different from original... I prefer TONE=3 but that's underexposed on the Gopro 11...

**Reply by dnewman-gpsw** [reply]
HERO12 or 13 is GPLog is slightly wider than WIDE=1.
